test/validation/api/scheduler/scheduler.c

```diff
@@ -795,21 +798,19 @@ static void scheduler_test_groups(void)
 		odp_schedule_group_leave(mygrp2, &mymask);
 		odp_schedule_group_join(mygrp1, &mymask);
 
+		wait_time = odp_schedule_wait_time(ODP_TIME_MSEC_IN_NS);
+		schedule_retries = 0;
 		while (num) {
 			queue = queue_grp[j];
-			ev    = odp_schedule(&from, ODP_SCHED_NO_WAIT);
+			ev    = odp_schedule(&from, wait_time);
 
 			if (ev == ODP_EVENT_INVALID) {
-				/* change group */
-				rc = odp_schedule_group_leave(mygrp[j],
-							      &mymask);
-				CU_ASSERT_FATAL(rc == 0);
-
-				j = (j + 1) % NUM_GROUPS;
-				rc = odp_schedule_group_join(mygrp[j],
-							     &mymask);
-				CU_ASSERT_FATAL(rc == 0);
+				CU_ASSERT_FATAL(schedule_retries !=
+							WAIT_1MS_RETRIES);
```

Comment:
I think `schedule_retries < WAIT_1MS_RETRIES` will be easier to read.

Reply:
Makes sense, will fix in V2.